torch/nn/modules/module.py

```diff
@@ -476,7 +476,7 @@ def forward(self, x):
     call_super_init: bool = False
     _compiled_call_impl: Optional[Callable] = None
 
-    def __init__(self, *args, **kwargs) -> None:
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
not sure it makes a difference but args could be a TypeVarTuple and kwargs could be Dict[str, Any]